### DISCUSSION TITLE
BarBot AI

### DISCUSSION THREAD

**Discussion by jtmuller5** [jtmuller5/The-HustleGPT-Challenge#31] [Ventures]
![image](https://user-images.githubusercontent.com/47997351/226369629-e5d05006-a9e8-4ae5-83be-301d3d47d930.png)

Website: [BarBot AI](https://apps.apple.com/us/app/barbot-ai/id1669668242)
Tagline: Bring the power of AI to your Home Bar
Twitter: [@JunaidDawud](https://twitter.com/JunaidDawud)

**Comment by jdawud** [comment]
Help an AI out and go download today!
